tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -299,6 +299,157 @@ def try_get_features():
                        timeout_msg="Timed out trying to get online feature values")
 
 
+@pytest.fixture(scope='module')
+def list_type_dataframe():
+    N_ROWS = 2
+    time_offset = datetime.utcnow().replace(tzinfo=pytz.utc)
+    customer_df = pd.DataFrame(
+        {
+            "datetime": [time_offset] * N_ROWS,
+            "customer_id2": [i for i in range(N_ROWS)],
+            "customer2_rating": [i for i in range(N_ROWS)],
+            "customer2_cost": [float(i)+0.5 for i in range(N_ROWS)],
+            "customer2_past_transactions_int": [[i,i+2] for i in range(N_ROWS)],
+            "customer2_past_transactions_double": [[float(i)+0.5,float(i)+2] for i in range(N_ROWS)],
+            "customer2_past_transactions_float": [[float(i)+0.5,float(i)+2] for i in range(N_ROWS)],
+            "customer2_past_transactions_string": [['first_'+str(i),'second_'+str(i)] for i in range(N_ROWS)],
+            "customer2_past_transactions_bool": [[True,False] for _ in range(N_ROWS)]
+        }
+    )
+    return customer_df
+
+@pytest.fixture(scope='module')
+def entity_list_type_dataframe():
```

Comment:
`list_type_dataframe` and `entity_list_type_dataframe` naming tell us nothing about what the dataframes are used for/meant to be ingested to etc.

Reply:
That's why I added "cases" in `test_basic_retrieve_online_dict`. But yup, I can be more descriptive with the dataframes, will add comments in those fixtures.